Packs/MicrosoftDefenderAdvancedThreatProtection/Integrations/MicrosoftDefenderAdvancedThreatProtection/MicrosoftDefenderAdvancedThreatProtection.py

```diff
@@ -4945,6 +4985,8 @@ def main():  # pragma: no cover
             return_results(tampering_command(client, args))
         elif command == 'microsoft-atp-advanced-hunting-cover-up':
             return_results(cover_up_command(client, args))
+        elif command == 'microsoft-atp-offboard-machine':
+            return_outputs(*offboard_machine_command(client, args))
```

Comment:
Why this changes are needed?
It's look like it change the original behavior

for example, how `command_status`will be `'Completed'`?